### PR TITLE
Feature/arts 301 site service add parent

### PR DESCRIPTION
**Pull request by kedarl-rdb**
## Description
Site Service Add Parent (ARTS-175)

### Changes
- ARTS-301 - Verify Organisation can connect to a parent. The Parent is optional.

### CODE REVIEW

**Comment by shajidissa** [comment]
Kedar, merge main into this and assign reviewers.

**Review by darrengsmithson — CHANGES_REQUESTED** [review]

File: api-test/config/conf.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+module.exports = {
+    BASE_URL: process.env.BASE_URL,
+    reporter: 'node_modules/mochawesome',
+    'reporter-option': [
+        'overwrite=true',
+        'reportTitle=My\ Custom\ Title',
+        'showPassed=false'
+    ],
+};
+
```
Comment (shajidissa) [inline api-test/config/conf.js:14]: All files in api-tests should not show in your PR as you've not added them. These should be what is in main branch.

File: site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/configuration/SiteAttribute.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import lombok.NoArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import lombok.AllArgsConstructor;
+
+/**
+ * Practitioner Attribute configuration Model
+ */
+@Component
+@Getter
```
[the remark this answers is not in the document]
Comment (shajidissa) [reply site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/configuration/SiteAttribute.java:15]: This is related to ARTS-376

File: site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/configuration/SiteConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+/**
+ * Site service configuration Model
+ */
+@Component
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+public class SiteConfiguration {
```
[the remark this answers is not in the document]
Comment (shajidissa) [reply site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/configuration/SiteConfiguration.java:20]: This is related to ARTS-376

File: site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/model/Site.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import lombok.Setter;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+/**
+ * Site Model
+ */
+@Data
+@Getter
```
[the remark this answers is not in the document]
Comment (shajidissa) [reply site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/model/Site.java:13]: This is related to ARTS-376

File: trial-config-service/Dockerfile [hunk 5]
```diff
@@ -6,6 +6,7 @@ COPY ./build-tools ./build-tools
 COPY ./pom.xml pom.xml
 
 # copy poms of ALL services
+COPY ./site-service/pom.xml ./site-service/pom.xml
 COPY ./trial-config-service/pom.xml ./trial-config-service/pom.xml
 COPY ./practitioner-service/pom.xml ./practitioner-service/pom.xml
 COPY ./sample-service/pom.xml ./sample-service/pom.xml
```
Comment (shajidissa) [inline trial-config-service/Dockerfile:12]: files in trial-config-service should not be part of your PR.

**Review by shajidissa — CHANGES_REQUESTED** [review]

**Comment by kedarl-rdb** [comment]
Closing this obsolete PR
New PR has been raised

**Comment by kedarl-rdb** [comment]
Closing this obsolete PR
New PR has been raised

**Pull request closed without merging**